ci/cron/daily-compat.yml

```diff
@@ -67,6 +67,27 @@ jobs:
           pathtoPublish: '$(Build.StagingDirectory)'
           artifactName: 'Bazel Compatibility Logs'
 
+  - job: platform-independence-windows
```

Comment:
Job names need to use underscores instead of dashes.

Reply:
fixed, thanks.